test/functional/cursor.test.js

```diff
@@ -2839,6 +2840,28 @@ describe('Cursor', function() {
     }
   });
 
+  /**
+   * @ignore
+   */
+  it('shouldSetReadConcern', {
+    metadata: {
+      requires: { topology: ['single', 'replicaset', 'sharded', 'ssl', 'heap', 'wiredtiger'] }
```

Comment:
if you want to test on all variants then there is no need to provide this `metadata` at all.

Reply:
cool also good to know.